gen/rewriter.jl

```diff
@@ -11,6 +14,11 @@ headerfiles = [
     "ogr_srs_api.jl"
 ]
 
+custom_rename = Dict{ASCIIString, ASCIIString}(
+    "gdalgetdriverbyname" => "gdalgetdriverbyname",
+    "ogrgetdriverbyname" => "ogrgetdriverbyname"
+)
+
```

Comment:
what is the purpose of `custom_rename`? I've only seen it used in `newfuncname` as

```julia
if name in keys(custom_rename)
    name = custom_rename[name]
else
    ...
```